src/app/appmarket/appinstance/appinstance/appinstance.component.html

```diff
@@ -134,12 +134,14 @@ <h3><em style="color: #337ab7;">{{ translateState(appInstanceStatus?.state) }}</
                                     </a>
                                 </li>
                                 <li *ngIf="!appInstance.autoUpgradesEnabled">
-                                    <a role="button" (click)="this.enableAutoUpgradesModal.show()">
+                                    <a *roles="['ROLE_DOMAIN_ADMIN', 'ROLE_SYSTEM_ADMIN']"
```

Comment:
@llopat Does `ROLE_VL_DOMAIN_ADMIN` should also be added here ?

Reply:
@kbeyro well probably yes, I assume there will be many such places ... but let's leave it for now